app/src/main/java/com/aurora/carevision/core/component/CVTopAppBar.kt

```diff
@@ -0,0 +1,152 @@
+package com.aurora.carevision.core.component
+
+import androidx.compose.foundation.Image
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.aurora.carevision.R
+import com.aurora.carevision.app.ui.theme.CVTheme
+import com.aurora.carevision.app.ui.theme.White
+
+@Composable
+fun TopAppBarBackLeft(
+    title: String,
+    LeftBackIcon: Int = R.drawable.ic_topappbar_leftbutton_24,
+    onBackClick: () -> Unit = {}
+) {
+    Row {
+        Box(
+            modifier = Modifier
+                .fillMaxWidth(),
+            //contentAlignment = Alignment.Center,
+            //horizontalArrangement = Arrangement.SpaceBetween
+        ){
+            IconButton(
+                onClick = { /*TODO*/ }
+            ) {
+                Image(
+                    painter = painterResource(id = LeftBackIcon),
+                    contentDescription = "LeftBack Icon",
+                    modifier = Modifier
+                        .size(36.dp)
+                        .padding(end = 12.dp)
+                )
+            }
+            Text(
+                text = title,
+                color = Color.Black,
+                style = CVTheme.typography.textBody1Importance,
+                textAlign = TextAlign.Center,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 12.dp)
+            ) // �� ģ���� ���Ϸε� ��� ��ġ��Ű�� �ͽ��ϴ� ->12.dp���� ���.. ��� �ϳ����
```

Comment:
```suggestion
@Composable
fun TopAppBarLeft(
    title: String = "",
    onBackClick: () -> Unit = {}
) {
    Row(
        horizontalArrangement = Arrangement.SpaceBetween,
        verticalAlignment = Alignment.CenterVertically,
        modifier = Modifier
            .fillMaxWidth()
            .background(White)
            .padding(vertical = 16.dp)
    ) {
        Box(
            modifier = Modifier.fillMaxWidth(),

            ) {
            Image(
                painter = painterResource(id = R.drawable.ic_topappbar_leftbutton_24),
                contentDescription = "LeftBack Icon",
                modifier = Modifier
            )
            Text(
                text = title,
                color = Color.Black,
                style = CVTheme.typography.textBody1Importance,
                textAlign = TextAlign.Center,
                modifier = Modifier
                    .fillMaxWidth()
                    .padding(start = 16.dp)
            )
        }
    }
}
```

이 컴포넌트(left) 이거 복붙해서 한번 보시고 이거 참고해서 아래것들도 수정해주세용

Reply:
이렇게 하면 해결됩니당